.changeset/tame-buckets-end.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-api-docs': minor
+'@backstage/plugin-techdocs': minor
+---
+
+Added filters prop to allow filter customization on both ApiExplorerPage and TechDocsIndexPage
```

Comment:
I think it would be nice to include a short snippet of code explaining how the new prop is used.